explorer/src/components/BondBreakdown.tsx

```diff
@@ -11,103 +11,144 @@ import Paper from '@mui/material/Paper';
-                        </TableRow>
-                        <TableRow>
-                            <TableCell
-                                sx={{
-                                    width: matches ? '90px' : 'auto',
-                                }}
-                                align='left'
-                            >
-                                Delegation total
-                            </TableCell>
-                            <TableCell align='left'>
-                                {bonds.delegations}{bonds.denom}
-                            </TableCell>
-                        </TableRow>
-                    </TableBody>
-                </Table>
+    React.useEffect(() => {
+        const hasError = Boolean(mixnodeDetailInfo?.error || delegations?.error);
+        const hasAllMixnodeInfo = Boolean(mixnodeDetailInfo?.data !== undefined && mixnodeDetailInfo?.data[0].mix_node);
+        const hasAllDelegationsInfo = Boolean(delegations?.data !== undefined && delegations?.data.length);
+        const hasAllData = Boolean(!hasError && hasAllMixnodeInfo && hasAllDelegationsInfo);
+        setShowError(hasError);
+        setAllContentLoaded(hasAllData);
+    }, [mixnodeDetailInfo, delegations]);
+
+    const calcBondPercentage = (num: number) => {
+        if (mixnodeDetailInfo?.data !== undefined && mixnodeDetailInfo?.data[0]) {
+            const rawDeligationAmount = Number(mixnodeDetailInfo.data[0].total_delegation.amount);
+            const rawPledgeAmount = Number(mixnodeDetailInfo.data[0].bond_amount.amount);
+            const rawTotalBondsAmount = rawDeligationAmount + rawPledgeAmount;
+            return (num * 100 / rawTotalBondsAmount).toFixed(1)
+        }
+    }
+
+    if (mixnodeDetailInfo?.isLoading) {
+        return <p>Loading...</p>
+    }
+
+    if (showError) {
+        return <p>We are unable to retrieve a Mixnode with that ID. Please try later or Contact Us.</p>
+    }
 
-                {delegations?.data !== undefined && delegations?.data[0] && (
-                    <Table sx={{ minWidth: 650 }} aria-label='delegation totals'>
-                        <TableHead>
+    if (!showError && allContentLoaded) {
+        return (
+            <>
+                <TableContainer component={Paper}>
+                    <Table sx={{ minWidth: 650 }} aria-label='bond breakdown totals'>
+                        <TableBody>
+                            <TableRow sx={matches ? { minWidth: '70vw' } : null}>
+                                <TableCell
+                                    sx={{
+                                        fontWeight: 'bold',
+                                        width: matches ? '90px' : 'auto',
+                                    }}
+                                    align='left'
+                                >
+                                    Bond total
+                                </TableCell>
+                                <TableCell align='left'>
```

Comment:
The formatting is still all over the place:

```
<TableCell align="left">
```

Are you letting eslint fix this, or are you searching and replacing?

Because you should let `eslint` fix it for you.